### ISSUE TITLE
Menu doesn't close if clicking out

### ISSUE THREAD

**Issue by carallo** [gonzaarcr/Fildem#19]
Sorry for bothering again!
I noticed a misbehaviour which I imagine you are aware of, so I'm not sure it's possible to fix. 
When you open a menu, the only way to have it close is by clicking again in the menu area, it doesn't close if you click anywhere else in the screen, as it's usual with app menus. Guess it's a limitation due to Gnome...

Anyway I'm using Fildem these days and I love it!

**Comment by gonzaarcr** [comment]
What OS are you using? And are you on Wayland?

**Comment by carallo** [comment]
> And are you on Wayland?
Yess, Wayland here!

**Comment by gonzaarcr** [comment]
Yeah, Wayland does have a problem with popups and floating stuff. Last time I checked, not event regular menus worked. Now they do, but I cannot drag a simple tab from sublime text out of the window.

I’m gonna see if I can talk with someone and see if they can tell me the state of the art of Wayland.

**Comment by carallo** [comment]
> 
> I’m gonna see if I can talk with someone and see if they can tell me the state of the art of Wayland.

No worries man, it's more than enough how it's working now for me! ;)

**Comment by KonstantMan** [comment]
I have the same problem in Arch with xorg , i have to click a menu item for hover to start working and then i have to click again  to close it

**Comment by igor-kobzev** [comment]
Great project, @gonzaarcr . I really wish the Wayland experience would get some love, now that most major distros are switching to Wayland even for Nvidia cards.

**Comment by ghost** [comment]
@Weather-OS this is a long-standing bug that's significantly plaguing the Wayland experience of using this software. It's what I was talking about in that other comment.

**Comment by Weather-OS** [comment]
@igor-kobzev I'll investigate this issue. If I had to guess, It is probably due to how Fildem is rendered with Xwayland.
